docs/source/process.rst

```diff
@@ -165,24 +138,31 @@ from the raw data file.
 However, since careful calibration is often done separately from the
 data collection phase of the field work, accurate calibration parameters
 are often supplied in the post-processing stage.
-Currently echopypy allows users to overwrite calibration parameters for
-EK60 data, including ``sa_correction``, ``equivalent_beam_angle``,
-and ``gain_correction``.
+Currently echopype allows users to overwrite the following calibration parameters:
+
+- EK60 and EK80: ``sa_correction``, ``gain_correction``, and ``equivalent_beam_angle``
+
+- AZFP: ``EL``, ``DS``, ``TVR``, ``VTX``, ``Sv_offset``, and ``equivalent_beam_angle``
+
 
-As an example, to reset the equivalent beam angle for 18 kHz only,
-one can do:
+As an example, to reset the equivalent beam angle for all frequencies,
+specify ``cal_params`` while calling the calibration functions like so:
 
 .. code-block:: python
 
-   ed.equivalent_beam_angle.loc[dict(frequency=18000)] = -18.02  # set value for 18 kHz only
+   # set all channels at once
+   calibration = {
+       'equivalent_beam_angle': [-17.47, -20.77, -21.13, -20.4 , -30]
+   }
+   Sv = ep.calibrate.compute_Sv(echodata, cal_params=calibration)
 
-To set the equivalent beam angle for all channels at once, do:
+To reset the equivalent beam angle for 18 kHz only, one can do:
 
 .. code-block:: python
 
-   ed.equivalent_beam_angle = [-17.47, -20.77, -21.13, -20.4 , -30]  # set all channels at once
+   echodata.beam.equivalent_beam_angle.loc[dict(frequency=18000)] = 18.02  # set value for 18 kHz only
 
-Make sure you use ``ed.equivalent_beam_angle.frequency`` to check
+Make sure you use ``echodata.beam.equivalent_beam_angle.frequency`` to check
 the sequence of the frequency channels first, and always double
 check after setting these parameters!
```

Comment:
The construction of the xr.DataArray from the above comment should allow removing this sentence all together. We actually need a check in the calibration function to take care of this, but let's delay that to later release.